package-lock.json

```diff
@@ -4547,98 +4547,6 @@
         "semver": "bin/semver.js"
       }
     },
-    "node_modules/@vue/compiler-core": {
```

Comment:
It looks like you rebuilt your lock file. Did you delete it and then run `npm i`?

It's probably a good thing to do that from time to time, and commit it. Otherwise you can have unused imports like these left over when a package is added or updated or deleted.

Reply:
Yes I did, I was also using node version 16.17.0 when I switched to 16.14.2 I got the same lock file as yours.

Reply:
That's funny. I deleted my lock file and reinstalled and I got a similar file as yours 😆 

If the node version makes a big difference then we should all plan to use the same version... maybe 16.17.0 ?

I'm not going to worry about this atm.